tests/drawing/test_draw.py

```diff
@@ -85,3 +87,55 @@ def test_color_arg_to_dict(edgelist4):
     assert np.allclose(d[1], np.array([[0.99692426, 0.89619377, 0.84890427, 1.0]]))
     assert np.allclose(d[2], np.array([[0.98357555, 0.41279508, 0.28835063, 1.0]]))
     assert np.allclose(d[3], np.array([[0.59461745, 0.0461361, 0.07558631, 1.0]]))
+
+
+def test_draw(edgelist8):
+
+    H = xgi.Hypergraph(edgelist8)
+
+    ax = xgi.draw(H)
+
+    assert len(ax.lines) == len(H.edges.filterby("size", 2))  # dyads
+    assert len(ax.patches) == len(H.edges.filterby("size", 2, mode="gt"))  # hyperedges
+    assert len(ax.collections[0].get_sizes()) == H.num_nodes  # nodes
+    plt.close()
+
+
+def test_draw_nodes(edgelist8):
```

Comment:
Can you add a test to make sure the lines, nodes, and hyperedges/simplices have the right zorder?

Reply:
Done